frontend/src/components/utils/table/sorting.ts

```diff
@@ -30,14 +30,34 @@ export const cycleSorting = (current?: Sorting) =>
 
 /**
  * Build an URL sorting parameter from a `sorting` object. The function iterates through the object's keys and returns
- * the sorting parameter corresponding to the _first_ key containing either `Sorting.Ascending` or `Sorting.Descending`
- * (in other words, multi-sort is _not_ supported).
+ * the value of the sorting parameter corresponding to the _first_ key containing either `Sorting.Ascending` or
+ * `Sorting.Descending` (in other words, multi-sort is _not_ supported).
+ *
+ * For instance:
+ * ```
+ * sortingToApiParameter(
+ *     {
+ *         myPiche: Sorting.Descending,
+ *         myPoche: Sorting.Ascending,
```

Comment:
is it a typo ?

Reply:
No, it's just for illustrating two different arguments

Reply:
Then maybe use more different words to let no ambiguity